renovate.json

```diff
@@ -18,7 +18,8 @@
       "postUpgradeTasks": {
         "fileFilters": ["magnum_cluster_api/manifests/calico/v*.yaml"],
         "commands": [
-          "curl -o magnum_cluster_api/manifests/calico/v{{{newValue}}}.yaml https://raw.githubusercontent.com/projectcalico/calico/v{{{newValue}}}/manifests/calico.yaml"
+          "curl -o magnum_cluster_api/manifests/calico/v{{{newValue}}}.yaml https://raw.githubusercontent.com/projectcalico/calico/v{{{newValue}}}/manifests/calico.yaml",
+          "sed -i 's|docker.io|quay.io|g' magnum_cluster_api/manifests/calico/v{{{newValue}}}.yaml"
```

Comment:
I think Renovate won't like this once we switch to the hosted application because the `postUpgradeTasks` does not support running `curl` commands.

Can you try to think of another way to try and update this?

Reply:
@mnaser post-upgrade tasks can only be used on self-hosted Renovate instances. Once we switch to hosted version, then we cannot use not only curl but also this whole post-upgrade.

Reply:
so i think it is impossible to patch manifests via renovate.